resources/acsl/digitalelectronics.md

```diff
@@ -9,12 +9,321 @@
 
 # Introduction
 
-Digital electronics is a way to visualize boolean algebra using drawn-out circuits.
+Digital electronics is a way to visualize boolean algebra using drawn-out circuits. The same laws of boolean algebra apply and truth tables are still very useful when analyzin them.
 
 <br>
 
 # Definitions
 
+<table class="wikitable" style="text-align: left">
```

Comment:
Why are you using HTML-style tables?

Use GitHub-Flavored Markdown's tables.

They look like this:

```
| something | something |
| --- | --- |
| Hi | Bye |
```

Which then transforms into this:
| something | something |
| --- | --- |
| Hi | Bye |

Reply:
Also, because this is Markdown, don't bother with HTML formatting for most things.
Bold is `**like this**`.